packages/manager/apps/pci-load-balancer/public/translations/common/Messages_pt_PT.json

```diff
@@ -0,0 +1,91 @@
+  "common_navbar_notification_mark_as_read": "Marcar como lida",
+  "common_navbar_notification_mark_as_unread": "Marcar como não lida",
+  "common_navbar_notification_none": "Está atualizado!",
+  "common_navbar_notification_none_description": "Não tem notificações.",
+  "common_open_new_tab": "Abre um novo separador",
+  "common_pagination_resultsperpage": "Resultados por página",
+  "common_pagination_page": "página",
+  "common_pagination_of": "em",
+  "common_pagination_ofnresults": "de TOTAL_ITEMS resultados",
+  "common_pagination_currentpageofpagecount": "Página CURRENT_PAGE de PAGE_COUNT",
+  "common_pagination_results": "resultados",
+  "common_pagination_next": "Página seguinte",
+  "common_pagination_previous": "Página anterior",
+  "common_choose": "Escolher",
+  "common_stepper_optional_label": "(opcional)",
+  "common_stepper_modify_this_step": "Editar este passo",
+  "common_stepper_skip_this_step": "Ignorar este passo",
+  "common_stepper_next_button_label": "Seguinte",
+  "common_stepper_submit_button_label": "Enviar",
+  "common_stepper_cancel_button_label": "Anular",
+  "common_file_attachmentsHeading": "Ficheiro(s) anexado(s)",
+  "common_file_dropArea": "Arraste e solte ficheiros ou",
+  "common_file_dropAreaSelector": "selecione um ficheiro",
+  "common_file_fileSelector": "Selecione um ficheiro",
+  "common_file_filesSelector": "Selecionar ficheiros...",
+  "common_file_maxsizeError": "O ficheiro ultrapassa o tamanho limite",
+  "common_file_removeFile": "Eliminar o ficheiro selecionado",
+  "common_password_all_rules_met": "Todas as condições de palavra-passe foram cumpridas.",
+  "common_password_hide_password": "Ocultar a palavra-passe",
+  "common_password_show_password": "Mostrar a palavra-passe",
+  "common_password_valid_rule": "Condição válida.",
+  "common_password_invalid_rule": "Condição inválida.",
+  "common_password_risky_password": "Palavra-passe de risco.",
+  "common_password_bad_password": "Palavra-passe incorreta.",
+  "common_password_weak_password": "Palavra-passe fraca.",
+  "common_password_good_password": "Palavra-passe correta.",
+  "common_password_strong_password": "Palavra-passe forte.",
+  "common_criteria_adder_filter_label": "Filtrar",
+  "common_back_button_back_to": "Voltar para",
+  "common_back_button_back_to_previous_page": "Voltar à página anterior",
+  "common_dual_list_source_heading": "Elementos por selecionar",
+  "common_dual_list_source_placeholder": "Nenhum elemento a selecionar",
+  "common_dual_list_source_move": "Adicionar",
+  "common_dual_list_source_move_all": "Adicionar tudo",
+  "common_dual_list_source_search": "Procurar no conteúdo de origem",
+  "common_dual_list_target_heading": "Elementos selecionados",
+  "common_dual_list_target_placeholder": "Nenhum elemento selecionado",
+  "common_dual_list_target_move": "Eliminar",
+  "common_dual_list_target_move_all": "Eliminar tudo",
+  "common_dual_list_target_search": "Procurar no conteúdo de destino"
+}
```

Comment:
The common translation are already in the pci-manager-common, u need just to use the namespace pci-common it work fine

Reply:
done thanks